client/src/components/highlightChild.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+class HighLightChild extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      view: 0
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    if (this.state.view === 0) {
+      this.setState({
+        view: 1
+      });
+    } else {
+      this.setState({
+        view: 0
+      });
+    }
+  }
+
+  render() {
+    if (this.state.view === 1) {
+      return (
+        <div onClick={this.handleClick}>
+          <div>
+            <span>Test * </span>
+            <span>{this.props.highlight}</span>
+          </div>
+        </div>
+      );
+    }
+    if (this.state.view === 0) {
```

Comment:
Continuing from my last comment, good place to enhance readability by using a constant name

Reply:
Refactored as explained above.